drivers/video/logo/logo.c

```diff
@@ -99,6 +99,10 @@ const struct linux_logo * __ref fb_find_logo(int depth)
 #ifdef CONFIG_LOGO_SUPERH_CLUT224
 		/* SuperH Linux logo */
 		logo = &logo_superh_clut224;
+#endif
+#ifdef CONFIG_LOGO_BRAIN_CLUT224
+              /* Generic Linux logo */
```

Comment:
(must) Generic Linux のロゴではないので表記を変えましょう。例えば以下のような感じで。

```suggestion
              /* Brain Hackers logo */
```